browser/brave_profile_prefs.cc

```diff
@@ -304,7 +304,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
 
   // Brave Today
   registry->RegisterDictionaryPref(kBraveTodaySources);
-  registry->RegisterBooleanPref(kBraveTodayIntroDismissed, false);
```

Comment:
Instead of deleting deprecated prefs, it would be good to have time to clear this prefs from user's prefs
See https://github.com/brave/brave-core/blob/master/chromium_src/chrome/browser/prefs/browser_prefs.cc#L37

Reply:
Fixed via https://github.com/brave/brave-core/compare/d30a27d6612be5d163ece3409db6012f3c32df98..f5160d8252b7ca157420992566c171cd44a1a30e